src/config.rs

```diff
@@ -142,3 +147,8 @@ fn test_alt_env_config_precedence() {
 	write("libreddit.toml", config_to_write).unwrap();
 	assert_eq!(get_setting("LIBREDDIT_DEFAULT_COMMENT_SORT"), Some("top".into()))
 }
+#[test]
+#[sealed_test(env = [("LIBREDDIT_DEFAULT_SUBSCRIPTIONS", "news+bestof")])]
+fn test_default_subscriptions() {
+	assert_eq!(get_setting("LIBREDDIT_DEFAULT_SUBSCRIPTIONS"), Some("news+bestof".into()));
```

Comment:
This appears to contradict the format of environment variable as given in the README, so I would have expected `["news", "bestof"]`.

Reply:
Excuse me for jumping in here. But i've asked to create this PR as i really enjoyed this option so far and not having this option around leaves anyone using this option kinda stuck on 0.26.0. It was the last version with it.

Having it `linux+gnome+news` was [how it worked before](https://github.com/libreddit/libreddit/issues/560). 

Not saying it wouldn't work as you suggest but it feels kinda odd when Libreddit itself already [supports this syntax](https://libreddit.spike.codes/r/linux+gnome+news) and also Reddit works this way [as you can see here](https://www.reddit.com/r/linux+gnome+news).

Reply:
@r7l: I agree that we should reintroduce this feature, but we need to make sure that our documentation is consistent and correct.

Reply:
@Daniel-Valentine I might have misunderstood your comment. Sorry. Seems like it just needs a last fix to the README file instead of a rewrite to the subscription list. Good to hear.

Reply:
:+1: Will fix the readme in the next commit.

Reply:
@r7l:

> @Daniel-Valentine I might have misunderstood your comment. Sorry. Seems like it just needs a last fix to the README file instead of a rewrite to the subscription list. Good to hear.

Not an issue. Just wanted to make sure we're all on the same page.